mobile/apps/photos/lib/services/smart_albums_service.dart

```diff
@@ -166,11 +167,14 @@ class SmartAlbumsService {
           newConfig = newConfig.addFiles(updatedAtMap, pendingSyncFiles);
 
           await saveConfig(newConfig);
-        } catch (_) {}
+        } catch (e, sT) {
+          _logger.warning(e, sT);
+        }
       }
     }
     syncingCollection = null;
     Bus.instance.fire(SmartAlbumSyncingEvent());
+    _logger.info("Smart Albums sync completed");
```

Comment:
We can this log or use fine? Just want to limit the number of logs in the log file to avoid redundant information.